test/unit/handlers/router-entities/route-caching/dynamo-route-caching-provider.test.ts

```diff
@@ -59,9 +59,9 @@ const TEST_WETH_USDC_POOL = new Pool(
   /* tickCurrent */ -69633
 )
 
-const TEST_USDC_UNI_POOL = new Pool(
```

Comment:
We will have to update these tests since they are not going to be maintanable in the future :p we don't need to do this now, but we will ahve to think more about it